src/main/java/ToDo.java

```diff
@@ -0,0 +1,12 @@
+public class ToDo extends Task {
+        public ToDo(String description) {
+            super(description);
+        }
```

Comment:
One excessive indentation. Curly brackets for ToDo class not aligned with "public"